src/react/src/components/ClaimAttachmentsUploader.jsx

```diff
@@ -1,15 +1,12 @@
 import React, { useState, useRef } from 'react';
 import { connect } from 'react-redux';
-import PropTypes from 'prop-types';
```

Comment:
PropTypes should be present there because you are passing props from parent `ClaimFacilitySupportDocs` component.